pkg/imgutil/imgutil.go

```diff
@@ -373,3 +387,61 @@ func ParseRepoTag(imgName string) (string, string) {
 
 	return repository, tag
 }
+
+func ParseFilters(filters []string) ([]string, []string, error) {
+	var beforeFilters []string
+	var afterFilters []string
+	for _, filter := range filters {
+		tempFilterToken := strings.Split(filter, "=")
+		switch len(tempFilterToken) {
+		case 1:
+			return nil, nil, fmt.Errorf("invalid filter %q", filter)
+		case 2:
+			if tempFilterToken[0] == "before" {
+				canonicalRef, err := referenceutil.ParseAny(tempFilterToken[1])
+				if err != nil {
+					return nil, nil, err
+				}
+				beforeFilters = append(beforeFilters, fmt.Sprintf("name==%s", canonicalRef.String()))
+				beforeFilters = append(beforeFilters, fmt.Sprintf("name==%s", tempFilterToken[1]))
+			} else if tempFilterToken[0] == "since" {
+				canonicalRef, err := referenceutil.ParseAny(tempFilterToken[1])
+				if err != nil {
+					return nil, nil, err
+				}
+				afterFilters = append(afterFilters, fmt.Sprintf("name==%s", canonicalRef.String()))
+				afterFilters = append(afterFilters, fmt.Sprintf("name==%s", tempFilterToken[1]))
+			} else {
+				return nil, nil, fmt.Errorf("invalid filter %q", filter)
+			}
+		default:
+			return nil, nil, fmt.Errorf("invalid filter %q", filter)
+		}
+	}
+	return beforeFilters, afterFilters, nil
+}
+
+func FilterImages(labelImages []images.Image, beforeImages []images.Image, afterImages []images.Image) []images.Image {
+	sort.Slice(beforeImages, func(i, j int) bool {
+		return beforeImages[i].CreatedAt.Before(beforeImages[j].CreatedAt)
+	})
+	sort.Slice(afterImages, func(i, j int) bool {
+		return afterImages[i].CreatedAt.Before(afterImages[j].CreatedAt)
+	})
+	var filteredImages []images.Image
+	maxTime := time.Now()
+	minTime := time.Date(1970, time.Month(1), 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Why is this required?

Reply:
A little trick, just make sure the logic match [minTime, maxTime] rules